js/DailyDouble.js

```diff
@@ -1,4 +1,4 @@
-const Clue = require('../js/Clue.js');
+// const Clue = require('../js/Clue.js');
```

Comment:
We still have to figure out how to get this line to work for Mocha while not causing errors in the browser.